docs/development.md

```diff
@@ -8,6 +8,37 @@ The source files are in the `/source` directory.  The `compile` rake task builds
 
 This resulting app directory is included in the gem and hooked in as a Rails engine
 
+## Getting a manual test rig setup
+
+You will need a 'test website' to manually verify changes. If you're only modifying the existing assets, you can do this with NodeJS.
+
+```
+npm install -g http-server node-sass
+
+# Serve all the assets in the current directory
+http-server -c-1
```

Comment:
You might want to use 'npx' so that people dont need to install http-server and node-sass globally.

Reply:
Cool - that's a new one for me.